frontend/src/components/Search/SearchForm/utils.ts

```diff
@@ -167,3 +178,79 @@ export const buildTokenGroups = (tokens: SearchToken[]) => {
 
 	return tokenGroups
 }
+
+export const useRetentionPresets = (productType: ProductType) => {
```

Comment:
looks like the `DateTimePicker` component still has a fixed time range for the UI - could we fix that too? 

![Screenshot from 2024-05-07 12-44-26](https://github.com/highlight/highlight/assets/1351531/bb4d0948-2cbe-40d6-9eda-ce0e231d1052)

![Screenshot from 2024-05-07 12-45-01](https://github.com/highlight/highlight/assets/1351531/dbeaf839-7785-4932-9e70-5970171c805f)

Reply:
was inconsistent - sometimes had the min date updated and sometimes not, seems to be fixed with a dependency upgrade